staging/src/k8s.io/client-go/util/workqueue/parallelizer.go

```diff
@@ -17,14 +17,15 @@ limitations under the License.
 package workqueue
 
 import (
+	"context"
 	"sync"
 
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 )
 
 type DoWorkPieceFunc func(piece int)
 
-// Parallelize is a very simple framework that allow for parallelizing
+// Parallelize is a very simple framework that allows for parallelizing
 // N independent pieces of work.
 func Parallelize(workers, pieces int, doWorkPiece DoWorkPieceFunc) {
```

Comment:
does it make sense to express this through the new one?

Reply:
What do you think it can be? I have no idea.

Reply:
Isn't `Parallelize(...) = ParallelizeUntil(nil, ...)`?

Reply:
Emmm, you can think of it this way, there will be a separate PR that will replace all Parallelize with ParallelizeUntil.
just like misterikkit said:https://github.com/kubernetes/kubernetes/pull/67555#discussion_r212155466

Reply:
@sttts The purpose of ParallelizeUntil is to allow the caller to control through the context. This PR is already using ParallelizeUntil.
The existence of Parallelize is temporary, as other packages are still using Parallelize. After this PR, I will submit a PR delete Parallelize separately.
I don't know if you understand what I mean. :)